skalko.oksana/homework_8/notification.rb

```diff
@@ -0,0 +1,48 @@
+module Notification
+  VALID_NUMBER = /^\+?3?8?(0[5-9][0-9]\d{7})$/.freeze
+  VALID_EMAIL = /\A[\w+\-.]+@[a-z\d\-]+(\.[a-z\d\-]+)*\.[a-z]+\z/i.freeze
+
+  def self.included(base)
+    base.extend(ClassMethods)
+  end
+
+  module ClassMethods
+    def log
+      if name.eql? 'Sms'
+        p File.read('Sms.log')
+      else
+        p File.read('Email.log')
+      end
+    end
+  end
+
+  def add_to_log(recepient)
```

Comment:
`File.open("#{self.class}.log", 'w') { |txt| txt.write(recepient) }`
same as in my comment above